### PR TITLE
Fixed python3 existence check

### PR DESCRIPTION
**Pull request by notstarboard**
This is to address Issue #954

### CODE REVIEW

File: monitoring/asgs-msgr.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ echo "RMQMessaging:Validating Message Service..."
 
 RMQMessaging_Python=python3 ; # should be using python provided by ASGS environment
 
-if [[ ! -e ${RMQMessaging_Python} ]] ; then 
+if [[ ! -e $(command -v ${RMQMessaging_Python}) ]] ; then
```
Comment (wwlwpd) [inline monitoring/asgs-msgr.sh:9]: I'd like use `which` here, mainly for consistency. But I also had to look up what `command -v` does, and it seems like `which` is what we want. 

But that does give me an idea for a `better` command that acts like `which` so I can say, "`which` is `better`" without it being a question. xD

... on second thought, change it also to use `-x` and wrap it in quotes, though it'd be strange for a platform to not have `python3` these days:

```
 if [[ ! -x "$(which ${RMQMessaging_Python})" ]] ; then 
```

This is untested, please make sure it works before you update the PR. Thanks!

**Review by wwlwpd — CHANGES_REQUESTED** [review]
see comment for requested change

**Comment by notstarboard** [comment]
which and -x seem to work just fine!

**Review by wwlwpd — APPROVED** [review]
Looks good to me!

**Comment by wwlwpd** [comment]
I approved it  but will merge it later since it has to be squashed.

**Comment by wwlwpd** [comment]
merged this with a `cherry-pick` externally due to _squash_

**Pull request closed without merging**